### PR TITLE
Added TRAM Metropolità d'Alcant

### PR DESCRIPTION
**Pull request by paunofu**
Hi, I have added the operator and network tags for the routes managed by the goverment company in the province of Alacant, Spain. It’s my first commit, I hope I don’t have many mistakes :)

### CODE REVIEW

File: data/transit/route/light_rail.json [hunk 1]
```diff
@@ -425,6 +425,20 @@
...
+        "matchNames": [ "TRAM", "TRAM Metropolitano de Alicante, TRAM Alicante" ],
+        "tags": {
+            "network": "TRAM Metropolità d'Alacant",
+            "network:wikidata": "Q2063567",
+            "network:wikipedia": "ca:TRAM Metropolità d'Alcant",
+            "operator": "Ferrocarrils de la Generalitat Valenciana",
+            "operator:wikidata": "Q750832",
+            "operator:wikipedia": "ca:Ferrocarrils de la Generalitat Valenciana",
+            "route": "light_rail"
+        },
```
Comment (LaoshuBaby) [inline data/transit/route/light_rail.json:441]:
```suggestion
          }
        },
```

File: data/transit/route/subway.json [hunk 2]
```diff
@@ -569,6 +569,20 @@
...
+        "matchNames": [ "TRAM", "TRAM Metropolitano de Alicante", "TRAM Alicante" ],
+        "tags": {
+            "network": "TRAM Metropolità d'Alacant",
+            "network:wikidata": "Q2063567",
+            "network:wikipedia": "ca:TRAM Metropolità d'Alcant",
+            "operator": "Ferrocarrils de la Generalitat Valenciana",
+            "operator:wikidata": "Q750832",
+            "operator:wikipedia": "ca:Ferrocarrils de la Generalitat Valenciana",
+            "route": "subway"
+        },
```
Comment (LaoshuBaby) [inline data/transit/route/subway.json:585]:
```suggestion 
     }  
   },
```

File: data/transit/route/train.json [hunk 3]
```diff
@@ -1998,6 +1998,20 @@
...
+        "matchNames": [ "TRAM", "TRAM Metropolitano de Alicante", "TRAM Alicante" ],
+        "tags": {
+            "network": "TRAM Metropolità d'Alacant",
+            "network:wikidata": "Q2063567",
+            "network:wikipedia": "ca:TRAM Metropolità d'Alcant",
+            "operator": "Ferrocarrils de la Generalitat Valenciana",
+            "operator:wikidata": "Q750832",
+            "operator:wikipedia": "ca:Ferrocarrils de la Generalitat Valenciana",
+            "route": "train"
+        },
```
Comment (LaoshuBaby) [inline data/transit/route/train.json:2014]:
```suggestion
       }
     },
```

File: data/transit/route/tram.json [hunk 4]
```diff
@@ -739,6 +739,20 @@
...
+        "matchNames": [ "TRAM", "TRAM Metropolitano de Alicante", "TRAM Alicante" ],
+        "tags": {
+            "network": "TRAM Metropolità d'Alacant",
+            "network:wikidata": "Q2063567",
+            "network:wikipedia": "ca:TRAM Metropolità d'Alcant",
+            "operator": "Ferrocarrils de la Generalitat Valenciana",
+            "operator:wikidata": "Q750832",
+            "operator:wikipedia": "ca:Ferrocarrils de la Generalitat Valenciana",
+            "route": "tram"
+        },
```
Comment (LaoshuBaby) [inline data/transit/route/tram.json:755]:
```suggestion
       }
    },
```

**Comment by LaoshuBaby** [comment]
Hello, friends! Welcome to NSI.

I just fix the typo error in json, and look the wikipedia, looks like there are only tram and train operated by Alicante TRAM, is it really need to create in subway and light_rail ?
Screenshot_2022-03-04-01-38-31-536_mark via

**Comment by paunofu** [comment]
The wikipedia is a bit outdated, the L9 is officially a train, but on ground is a light train and the L4 is a tram and L2 a metro. And you are right, I have just checked and in Barcelona they also use tram.

**Comment by paunofu** [comment]
It could a possibility change the ISO of the country (ES) for the ISO of the province (ES-A)?

Because the 90% use only TRAM instead of another denomination.

**Comment by LaoshuBaby** [comment]
> It could a possibility change the ISO of the country (ES) for the ISO of the province (ES-A)?

Sorry, location-conflation didn't support this currently.

图片

Only those area have a geojson can use provincial name, such as `us-mi.geojson`, you can see them in this folder:

https://github.com/osmlab/name-suggestion-index/tree/main/features/us

图片

---

But don't worry, you can use a circle region and add a note to tell other where the region affect:

图片

(You can have a try at: https://location-conflation.com/?locationSet=%7B%20include%3A%20%5B%5B-0.4881%2C%2038.3436%2C%2025%5D%5D%20%7D)

**Comment by paunofu** [comment]
I have the geojson of three regions of Spain, one of them the region that I want, I can upload it to features.

**Review by arch0345 — CHANGES_REQUESTED** [review]
Thanks for adding these geojsons, however, NSI [only supports](https://github.com/osmlab/name-suggestion-index/wiki/Feature-Files) a single geojson `Feature` (or a FeatureCollection containing a single `Feature`) and `Polygon` or `MultiPolygon` geometries. I recommend creating these by hand instead with geojson.io using the polygon tool as suggested in the wiki article linked above. We try to use as few points as possible, see https://github.com/osmlab/name-suggestion-index/issues/4784#issuecomment-742003434. It looks like others use less than 100 points.

**Review by arch0345 — APPROVED** [review]
Thanks for your patience @paunofu! This looks good to me

**Pull request merged**